README.md

```diff
@@ -94,6 +104,17 @@ This will plan a deployment, but not execute it -- useful to see ahead what chan
 
 The big kahuna! This will deploy all of your changes, including redeploying lambdas if any of code changes.
 
+## Query
+
+#### `./indexer query <CID>`
+Attempts to find the given CID in the Indexer node. The result is a Location Claim that needs to be used to fetch the actual content associated with that CID. In case you want to query a specific node, you can use the following command:
+
+```sh
+./indexer query -u https://<NODE_NAME>.indexer.storacha.network <CID>
```

Comment:
to keep the docs generic, I'd say something like:
```suggestion
./indexer query -u https://<INDEXING_SERVICE_URL> <CID>
```

if someone external to the company is deploying its own copy they will have their own URLs, or even IPs